recipes/glibmm/all/conanfile.py

```diff
@@ -43,129 +51,115 @@ def _giomm_lib(self):
         return f"giomm-{self._abi_version}"
 
     def validate(self):
-        if hasattr(self, "settings_build") and build.cross_building(self):
+        if hasattr(self, "settings_build") and cross_building(self):
             raise ConanInvalidConfiguration("Cross-building not implemented")
```

Comment:
It should work now with MesonToolchain. It was a limitation of the legacy Meson build helper.